failing_datasets/2024-01-17 15_13_45 ec-included-mpt-missing/2024-01-17 15_13_45 ec-included-mpt-missing.tsv

```diff
@@ -0,0 +1,663 @@
+subscribe	Whether to send events out on the web API for this channel	C4M32	bool	true
+color	color	C4M32	int	0
+mass	Measurement mass	C5M40	double	40.0
+lead_in_multiplier	Lead in multiplier	C5M40	int	1
+dwell_time	Dwell time [ms]	C5M40	int	32
+label_suffix	Label suffix (added to the mass to form the full label)	C5M40	string	Ar
+subscribe	Whether to send events out on the web API for this channel	C5M40	bool	true
+color	color	C5M40	int	1556175
+mass	Measurement mass	C6M44	double	44.0
+lead_in_multiplier	Lead in multiplier	C6M44	int	1
+dwell_time	Dwell time [ms]	C6M44	int	32
+label_suffix	Label suffix (added to the mass to form the full label)	C6M44	string	CO2
+subscribe	Whether to send events out on the web API for this channel	C6M44	bool	true
+color	color	C6M44	int	14034728
+data_dir	Data directory		string	C:\Users\Spectro Inlets\Spectro Inlets Dropbox\03 LGA\In-line MS\Chip testing on EC-MS setup
+project_name	Project name		string	failing_datasets
+measurement_name	Measurement name		string	ec-included-mpt-missing
+comment	Comment		string	Normal EC-MS chip
+ms_scan_interval	Mass spec scan interval [ms]		int	0
+ms_lead_in_base	Mass spec lead in base [ms]		int	32
+start_time_unix	Start time as UNIX time stamp		double	1705500829.614609
+start_timestamp	Start time in ISO 8601 format		string	2024-01-17 15:13:49.614609
+offset	The offset (time - EC-Lab time)	pot_01	double	9.326230e+00
+offset_meas_time	Offset measurement time [s]	pot_01	double	3.058147e-02
+Iongauge value_count	Number of points in the "Iongauge value" data series	Iongauge value	int	191
+Pirani1 value_count	Number of points in the "Pirani1 value" data series	Pirani1 value	int	191
+Pirani2 value_count	Number of points in the "Pirani2 value" data series	Pirani2 value	int	191
+Pirani3 value_count	Number of points in the "Pirani3 value" data series	Pirani3 value	int	191
+MFC1 value_count	Number of points in the "MFC1 value" data series	MFC1 value	int	191
+MFC1 setpoint_count	Number of points in the "MFC1 setpoint" data series	MFC1 setpoint	int	191
+MFC2 value_count	Number of points in the "MFC2 value" data series	MFC2 value	int	191
+MFC2 setpoint_count	Number of points in the "MFC2 setpoint" data series	MFC2 setpoint	int	191
+PC value_count	Number of points in the "PC value" data series	PC value	int	191
+PC setpoint_count	Number of points in the "PC setpoint" data series	PC setpoint	int	191
+C0M2_count	Number of points in the "C0M2" data series	C0M2	int	140
+C1M4_count	Number of points in the "C1M4" data series	C1M4	int	140
+C2M18_count	Number of points in the "C2M18" data series	C2M18	int	140
+C3M28_count	Number of points in the "C3M28" data series	C3M28	int	140
+C4M32_count	Number of points in the "C4M32" data series	C4M32	int	140
+C5M40_count	Number of points in the "C5M40" data series	C5M40	int	140
+C6M44_count	Number of points in the "C6M44" data series	C6M44	int	140
+pot_count	Number of points in the "pot" data series	pot	int	544
```

Comment:
In regards to the other PR, this is what I was referring to. Here it is simply called "pot_count". Have you got that detail under control?

Reply:
I do. In datasets it is only "pot_count". Ixdat then makes it "pot_pot_count" during runtime in order to keep unique series names.
https://github.com/ixdat/ixdat/blob/e66675fc4aa81c75d6e857983e5a87434c042786/src/ixdat/readers/zilien.py#L64